components/cart-summary/index.js

```diff
@@ -1,40 +1,73 @@
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import colors from '../../color/color.json';
+import Modal from '../Modal/index';
+import classNames from '../Modal/modal.module.css';
```

Comment:
Please arrange imports
Usually we have the library import first like react etc , then user module import and at last css imports

Reply:
@Kratika0907 Changes has been made please check.